src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,212 @@
+        System.out.println(Messages.LINE_DIVIDER);
+        System.out.println(Messages.LIST_TASKS_MESSAGE);
+        for (int i = 0; i < Parser.getOrderAdded() - 1; i++) {
+            recordedTask.get(i).printEntireTaskList();
+        }
+        System.out.println(Messages.LINE_DIVIDER);
+    }
+
+    private static void setTaskAsDone(String commandEntered, ArrayList<Task> recordedTask) {
+        try {
+            if (!commandEntered.substring(6).equals("")) {
+                throw new DukeException();
+            }
+        } catch (DukeException de) {
+            printUnrecognisedCommandErrorMessage();
+            return;
+        }
+        char taskNumToCheck = commandEntered.charAt(5);
+        int taskNumToChange = Character.getNumericValue(taskNumToCheck);
+        recordedTask.get(taskNumToChange - 1).setTaskStatus(true);
+        System.out.println(Messages.LINE_DIVIDER);
+        System.out.println(Messages.MARKED_TASKS_DONE_MESSAGE);
+        recordedTask.get(taskNumToChange - 1).markedAsDone();
+        System.out.println(Messages.LINE_DIVIDER);
+    }
+
+    private static void printByeMessage() {
+        System.out.println(Messages.LINE_DIVIDER);
+        System.out.println(Messages.BYE_MESSAGE);
+        System.out.println(Messages.LINE_DIVIDER);
+    }
+
+    protected static void deleteTask(String commandEntered, ArrayList<Task> recordedTask) {
+        try {
+            if (!commandEntered.substring(8).equals("")) {
+                throw new DukeException();
+            }
+        } catch (DukeException de) {
+            printUnrecognisedCommandErrorMessage();
+            return;
+        }
+        char taskNumToCheck = commandEntered.charAt(7);
+        int taskNumToChange = Character.getNumericValue(taskNumToCheck);
+        System.out.println(Messages.LINE_DIVIDER + "\n" + Messages.DELETE_TASK_STATEMENT);
+        recordedTask.get(taskNumToChange - 1).printTaskListing();
+        for (int i = taskNumToChange; i < Parser.getOrderAdded() - 1; i++) {
+            recordedTask.get(i - 1).setTaskNum(i);
+        }
+        orderSubtractor();
+        System.out.println(Messages.DELETE_TASK_DECLARATION + (Parser.getOrderAdded() - 1) + Messages.DELETE_TASK_STATEMENT_END);
+        System.out.println(Messages.LINE_DIVIDER);
+        recordedTask.remove(recordedTask.get(taskNumToChange - 1));
+        String updatedText = "";
+        for (int i = 0; i < orderAdded - 1; i++) {
+            updatedText = updatedText.concat(recordedTask.get(i).getCurrentTaskType()
+                    + " | " + recordedTask.get(i).taskStatus()
+                    + " | " + recordedTask.get(i).getTaskName()) + "\n";
+        }
+        try {
+            Save.writeToFile(Messages.SAVE_FILE_PATH, updatedText);
+        } catch (IOException ioe) {
+            System.out.println(Messages.IOEXCEPTION_ERROR);
+        }
+    }
+
+    protected static void orderAdder() {
```

Comment:
An adder is classified as a noun, perhaps the verb form "add" would be a better choice to represent this method? eg. addOrder or something similar